src/user/room.ts

```diff
@@ -200,6 +212,20 @@ export class Room extends Accessory implements Sayable {
    * @private
    * About the Generic: https://stackoverflow.com/q/43003970/1123955
    */
+  /**
+   * Load room by topic. <br>
+   * > Tips: For Web solution, it cannot get the unique topic id,
+   * but for other solutions besides web,
+   * we can get unique and permanent topic id.
```

Comment:
Add a link to explain this with more examples.

For example, add an entrance to the FAQ.